thirdparty/build-thirdparty.sh

```diff
@@ -734,6 +734,15 @@ build_aliyun_oss_jars() {
     cp -r $TP_SOURCE_DIR/$ALIYUN_OSS_JARS_SOURCE $TP_INSTALL_DIR/aliyun_oss_jars
 }
 
+build_aws_cpp_sdk() {
+    check_if_source_exist $AWS_SDK_CPP_SOURCE
+    cd $TP_SOURCE_DIR/$AWS_SDK_CPP_SOURCE
+    # only build s3 and s3-crt, you can add more components if you want.
+    $CMAKE_CMD -Bbuild -DBUILD_ONLY="core;s3;s3-crt" -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=${TP_INSTALL_DIR} -DENABLE_TESTING=OFF
```

Comment:
What's the difference between s3 and s3-crt?

Reply:
s3 is a standard s3 client, and s3-crt is the optimized version. 

https://github.com/aws/aws-sdk-cpp/wiki/Improving-S3-Throughput-with-AWS-SDK-for-CPP-v1.9

![image](https://user-images.githubusercontent.com/1081215/150628757-3594435e-b56a-4b0e-98b5-84a10be9bfde.png)

Reply:
please note that s3-crt is not stable according to my test.